### PR TITLE
Added xfstests test in avocado[V3]

### PR DESCRIPTION
**Pull request by PraveenPenguin**
Signed-off-by: Praveen K Pandey praveen@linux.vnet.ibm.com
v1 #105
v2 #143 

```
v3 : change git repo pointing to git://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git as oss git moved 
v3: checked in modified README file 
```

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
@apahim please find job log in below link

https://paste.fedoraproject.org/409723/14474711/

File: fs/xfstests.py.data/README [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+     ex : export TEST_DEV =/dev/vda1 , export TEST_DIR=/mnt
+
+     DO NOT USE A BLOCK DEVICE WITH IMPORTANT DATA!!!
+
+
+1.2) In yaml file set test_number which test need to run  and skip_dangerous
+     (it will check with group ) and    will skip all the test .
+
+1.3) 
+    ------TODO---------------------------------------------------------------
```
Comment (apahim) [inline fs/xfstests.py.data/README:19]: This section is still not clear. I can't say what I have to do with this TEST RANGE neither why there's a `-----TODO-----` line. Also, still can't say why I have to care about python's `range()`.

File: fs/xfstests.py.data/README [hunk 2]
```diff
@@ -0,0 +1,49 @@
+This is a simple wrapper for running xfstests inside avocado. The steps to get
+started are really simple:
+
+1) Edit the configuration variables on the yaml file as well environment variable
+
+1.1) The variables 'TEST_DEV' and 'TEST_DIR' are mandatory and should be set to
+     a block device path and mount point path, respectively, that will be used
+     *exclusively* for xfstests. It must have the filesystem of your choice
+     previously created.
+     ex : export TEST_DEV =/dev/vda1 , export TEST_DIR=/mnt
```
Comment (apahim) [inline fs/xfstests.py.data/README:10]: While on it, please fix the white-spaces here ^

File: fs/xfstests.py.data/README [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+     DO NOT USE A BLOCK DEVICE WITH IMPORTANT DATA!!!
+
+
+1.2) In yaml file set test_number which test need to run  and skip_dangerous
+     (it will check with group ) and    will skip all the test .
```
Comment (apahim) [inline fs/xfstests.py.data/README:16]: ... and here ^

File: fs/xfstests.py.data/README (hunk 1 above)
Comment (PraveenPenguin) [inline fs/xfstests.py.data/README:19]: @apahim thanks for looking into this ,proposal like this when user want to select run more than one test case TEST RANGE environment variable or variable in yaml file will help  but still this is TODO, if you agree than i can remove this section .

File: fs/xfstests.py.data/README (hunk 3 above)
Comment (PraveenPenguin) [inline fs/xfstests.py.data/README:16]: sure will fix and raise a new PR

File: fs/xfstests.py.data/README (hunk 1 above)
Comment (apahim) [inline fs/xfstests.py.data/README:19]: I'm ok with removing. It's your test :)

**Comment by apahim** [comment]
This repository is frozen. Please send the PR with the requested changes to:
https://github.com/avocado-framework-tests/avocado-misc-tests

**Pull request closed without merging**